bindings/python/src/alert.cpp

```diff
@@ -1029,6 +1057,27 @@ void bind_alert()
         .add_property("dropped_alerts", &get_dropped_alerts)
         ;
 
+    class_<dht_live_nodes_alert, bases<alert>, noncopyable>(
+       "dht_live_nodes_alert", no_init)
+        .add_property("node_id", &dht_live_nodes_alert::node_id)
+        .add_property("num_nodes", &dht_live_nodes_alert::num_nodes)
+        .add_property("nodes", &dht_live_nodes_nodes)
+        ;
+
+    std::vector<sha1_hash> (dht_sample_infohashes_alert::*samples)() const = &dht_sample_infohashes_alert::samples;
+
+    class_<dht_sample_infohashes_alert, bases<alert>, noncopyable>(
+       "dht_sample_infohashes_alert", no_init)
+        //.add_property("endpoint", &dht_sample_infohashes_alert::endpoint)
+        .add_property("endpoint", make_getter(&dht_sample_infohashes_alert::endpoint, by_value())) // TODO: should be correct like this, but in Python endpoint contains only garbage
+        //.add_property("interval", &dht_sample_infohashes_alert::interval) // TODO: throws error in Python; there is a converter for 'time_duration' in 'datatime.cpp', so this should work, but doesn't; type/class: dht_sample_infohashes_alert::interval == time_duration == std::chrono::nanoseconds
```

Comment:
iirc, in order to invoke the converter, you have to expose this field like this:
`.add_property("interval", make_getter(&dht_sample_infohashes_alert::interval), by_value())`

Reply:
would you mind testing this change?

Reply:
works!